src/components/datepicker/datePicker.js

```diff
@@ -179,11 +202,23 @@
     this.ngModelCtrl = null;
 
     /** @type {HTMLInputElement} */
-    this.inputElement = $element[0].querySelector('input');
+    this.inputElement = $element[0].querySelector('input.begin');
 
     /** @final {!angular.JQLite} */
     this.ngInputElement = angular.element(this.inputElement);
 
+    /** @type {HTMLInputElement} */
+    this.beginInputElement = $element[0].querySelector('input.begin');
+
+    /** @final {!angular.JQLite} */
+    this.ngBeginInputElement = angular.element(this.beginInputElement);
```

Comment:
Do you really need both the native and the wrapped elements? I'd prefer to just use the native elements throughout the code.